ci/release-info-v2.py

```diff
@@ -0,0 +1,96 @@
+import json
+import os
+import re
+import sys
+from datetime import date
+from subprocess import Popen, PIPE
+
+def read_version(crate):
+    # Reads package version from workspace manifest file
+    channel = Popen(
+        ['cargo', 'read-manifest', '--manifest-path', './{}/Cargo.toml'.format(crate)],
+        stdout=PIPE
+    )
+
+    # parse json data
+    data = json.load(channel.stdout)
+    version = data['version']
+    return version
+
+def check_version(ver):
+    # Checks package version for matching with the current tag reference
+    if ref is not None and ref != 'refs/tags/v' + str(ver):
+        return 0
+    else:
+        return 1
+
+def print_error(crate, ver, v):
+    # print(crate, ver, v)
+    if not v:
+        print(
+            '::error file={path}::version {0} does not match release tag {1}'
```

Comment:
`f'::error file={str(crate)}/Cargo.toml::version {str(ver)} does not match release tag {ref}'`

Reply:
I understand this relative new approach in Python for forming the strings, but this involve more complication and decrease the readability, so I leave this as it was in original script, just added typecast to string for version.